packages/veritone-react-common/src/components/Image/index.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Image from './';
+
+describe('Image', () => {
+  const TEST_IMG = 'https://pp.userapi.com/c624631/v624631866/38602/miSZV1RlzEY.jpg';
+  
+  it('Image should render in a div tag', () => {
```

Comment:
can you modify these to conform to a sentence, i.e 

`it('should render the image in a div tag')`
`it('should have a default height/width')`
`it('allows for height/width to be specified')`
`it('allows for a border to be set')`

Reply:
yep, i sure can